proto/cosmos/base/abci/v1beta1/abci.proto

```diff
@@ -48,6 +48,13 @@ message TxResponse {
   repeated tendermint.abci.Event events = 13 [(gogoproto.nullable) = false];
 }
 
+// BlockResponse placeholder - still needs work
```

Comment:
Simple obj - still need to figure out how to map the complex Block response in proto